src/Middleware.php

```diff
@@ -57,6 +59,8 @@ public function handle(Request $request, Closure $next)
 
         Inertia::share($this->share($request));
 
+        Inertia::setRootView($this->rootView($request) ?? $this->rootView);
```

Comment:
@reinink I just checked, but this would be a breaking change for PHP <= 7. 

I already made a PR earlier ([testing & CI pipeline](https://github.com/inertiajs/inertia-laravel/pull/132)) in which I basically came to the conclusion that if we want full test coverage, we can't really support PHP 7.1 and earlier anyway, so I don't think this is really a big issue. Thoughts?

Reply:
I'd say drop anything lower than that. Its actually one of Spatie's standards to drop older versions rather quickly.

Reply:
@Cannonb4ll Hm, actually, I think we can avoid this breaking change entirely by returning `$this->rootView` from the `rootView()` method by default:
```suggestion
        Inertia::setRootView($this->rootView($request));
```

Reply:
I've tried that indeed, but then you'd need to remove it from the stub as it would return`null` otherwise. Certainly possible.

Let's wait what @reinink thinks?

Reply:
Yes, sure :) 

One thing I'd like to add: Personally, I think I'd more often than not use the `prop` instead of the entire method to set the root view.. It's a bit shorter/simpler, and like I already mentioned on Discord I personally haven't ever had the need to use logic to determine the root view, and unless I'm wrong, I think the same would apply to most other situations?

Reply:
@Cannonb4ll I just briefly discussed this with @reinink, and he suggested to do pretty much exactly what I mentioned previously. I've implemented those changes, as well as added the tests that I mentioned I would add before.